config/crd/bases/ferry.zsm.io_clusterinformations.yaml

```diff
@@ -38,47 +38,209 @@ spec:
             description: ClusterInformationSpec defines the desired state of ClusterInformation
             properties:
               egress:
```

Comment:
provide an example.yaml would be better?

Reply:
Added